### ISSUE TITLE
default block props is start

### ISSUE THREAD

**Issue by mteason** [dominikbulaj/react-scroll-into-view#21]
if alignToTop is default, default of props block is also start.

**Comment by dominikbulaj** [comment]
Hi,
not sure what you mean, but yes, when `alignToTop` is set to `true` then those two options are used:

```javascript
if (alignToTop) {
  options.block = 'start' // defines vertical alignment 
  options.inline = 'nearest' // defines horizontal alignment
}
```

Ref.: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView

---
If you think it's an issue, feel free to describe proper behavior :)

**Comment by mteason** [comment]
sorry for my bad english,
I just want to set block to 'center'

**Comment by dominikbulaj** [comment]
So you would like to set both `block` and `inline` to `center`, right? I think new prop can be used for that - something like `alignCenter : boolean`

**Comment by dominikbulaj** [comment]
Released as v1.9.0
You can pass `scrollIOptions` that accepts an object with the same properties as you can pass to `element.scrollIntoView(scrollIntoViewOptions);`. See [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#parameters)
